style.css

```diff
@@ -96,8 +97,53 @@ body {
   min-height: 80px;
   border: none;
   border-radius: 4px;
+  outline: 0;
 }
 
+.quiz-option-button:hover {
+  color: #bb0a1e;
+  transition:         0.08s ease-in;
+  -o-transition:      0.08s ease-in;
+  -ms-transition:     0.08s ease-in;
+  -moz-transition:    0.08s ease-in;
+  -webkit-transition: 0.08s ease-in;
+}
+
+.game-restart-button {
+  cursor: url(/assets/knife.cur), pointer;  
+  position: relative;
+  color: #ffffff;
+  border: 2px #ffffff solid;
+  text-transform: uppercase;
+  background: none;
+  z-index: 1;
+  transition:         0.08s ease-in;
+  -o-transition:      0.08s ease-in;
+  -ms-transition:     0.08s ease-in;
+  -moz-transition:    0.08s ease-in;
+  -webkit-transition: 0.08s ease-in;
+}
+
+.fill:hover {
+  color: rgb(29, 134, 29);
+}
+
+.fill:before {
```

Comment:
Love the effect. If you make it fall from top to bottom like a [guillotine](https://www.google.com/search?client=firefox-b-d&q=guillotine) it would look more cool.